exercises/exercise3.js

```diff
@@ -8,5 +8,145 @@ module.exports.botScripts = [
+    isListening: true,
+  },
+  {
+    //This will return the users status if he is online or not
+    label: 'Whats my status',
+    prompt: 'Am offline or online?',
+    handler: function () {
+      const onlineStatus = 20;
+      const offlineStatus = 75;
+      const num = 50;
+      if (onlineStatus < 50) {
+        return 'You are online!';
+      } else if (offlineStatus > 50) {
+        return 'You are offline!';
+      } else {
+        return 'You need better internet.....';
+      }
+    },
+    isCaseSensitive: false,
+    isListening: true,
+  },
+  {
+    label: 'This is a magic eight ball buddy',
+    prompt: 'My name is #{userName}, whats my destiny',
+    handler: function (params) {
+      let userName = params.userName;
+      userName
+        ? console.log('Hello,' + userName + ' !')
+        : console.log('Hello!');
+      const randomNumber = Math.floor(Math.random() * 8);
+      let eightBall = '';
+      switch (randomNumber) {
+        case 0:
+          eightBall = 'Reconsider this question';
+          break;
+        case 1:
+          eightBall = 'Here we go agian...';
+          break;
+        case 2:
+          eightball = 'Really dude? why... like why...';
+          break;
+        case 3:
+          eightBall = 'I dont get paid enough for this';
+          break;
+        case 4:
+          eightBall = 'Its not what your doing right now bud.';
+          break;
+        case 5:
+          eightBall = 'Eh, maybe you have some purpose';
+          break;
+        case 6:
+          eightBall = 'You can be a sailor';
+          break;
+        case 7:
+          eightBall = 'Just be a programmer and live happy!';
+          break;
+      }
+      console.log(eightBall);
+    },
+    isCaseSensitive: false,
+    isListening: true,
+    isreply: true,
+  },
+  {
+    label: 'What is 2 + 2',
+    prompt: ' whats #{number} + #{number1}',
+    handler: function (params) {
+      if (params.number === '' && params.number1 === '') {
+        return '';
+      } else {
+        return null;
+      }
+    },
+    isCaseSensitive: false,
+    isListening: true,
+    isreply: true,
+  },
+  {
+    label: 'Hard day',
+    prompt: 'today was rough',
+    handler: function () {
+      return 'im sorry, would you like to talk about it?';
+    },
+
+    isCaseSensitive: false,
+    isListening: true,
+    isreply: true,
+  },
+  {
+    label:
+      'Its race day! If you registered at a certain time come grab your time and number!',
+    prompt: 'what time do i race and whats my number?',
+    handler: function () {
+      let raceNumber = Math.floor(Math.random() * 1000);
+      let registeredEarly = true;
+      let runnersAge = '18';
+      if ((runnersAge += '18' && registeredEarly)) {
+        console.log(
+          'Your number is' +
+            ' ' +
+            raceNumber +
+            '. ' +
+            'You may start your engines!!!!!'
+        );
+      } else if ((runnersAge += '18' && !registeredEarly)) {
```

Comment:
On line 121 and 129, you have "runnersAge += '18'" I think what you meant was '==' or '>' or '<' as you are checking to see if the runners age is over 18? Either way, if you use '+=' you will add '18' to the runnersAge instead, not check for anything.